src/graph/FetchEdgesExecutor.cpp

```diff
@@ -31,7 +30,7 @@ Status FetchEdgesExecutor::prepareClauses() {
         expCtx_ = std::make_unique<ExpressionContext>();
         expCtx_->setStorageClient(ectx()->getStorageClient());
         spaceId_ = ectx()->rctx()->session()->space();
-        yieldClause_ = sentence_->yieldClause();
+        yieldClause_ = DCHECK_NOTNULL(sentence_)->yieldClause();
```

Comment:
AFAIK. `DCHECK_*` expands to empty if built in release mode.

Reply:
DCHECK_NOTNULL return the expression value in release building.

Reply:
This PR is built ok in release mode.